Product/SoapUI_Test/RegressionSuite/Passthru/pom.xml

```diff
@@ -22,7 +22,6 @@
             </activation>
             <modules>
                 <module>Admin-Distribution-Passthrough</module>
-                <module>AuditLogging-Passthrough</module>
```

Comment:
Since the module still exists, and will be brought back at some point, I suggest commenting this line out instead of removing it.